keras_cv/models/backbones/densenet/densenet_backbone_presets_test.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for loading pretrained model presets."""
+
+import pytest
+import tensorflow as tf
+from absl.testing import parameterized
+
+from keras_cv.models.backbones.densenet.densenet_backbone import (
+    DenseNet121Backbone,
+)
+from keras_cv.models.backbones.densenet.densenet_backbone import (
+    DenseNetBackbone,
+)
+
+
+@pytest.mark.large
+class DenseNetPresetSmokeTest(tf.test.TestCase, parameterized.TestCase):
+    """
+    A smoke test for DenseNet presets we run continuously.
+    This only tests the smallest weights we have available. Run with:
+    `pytest keras_cv/models/backbones/densenet/densenet_backbone_presets_test.py --run_large`  # noqa: E501
+    """
+
+    def setUp(self):
+        self.input_batch = tf.ones(shape=(2, 224, 224, 3))
+
+    @parameterized.named_parameters(
+        ("preset_with_weights", "densenet121_imagenet"),
+        # ("preset_no_weights", "densenet121"),
```

Comment:
Just remove the parameterization entirely as in MobileNet ([link](https://github.com/keras-team/keras-cv/blob/b514c84650bc2851826e63328a6185077d7c747b/keras_cv/models/backbones/mobilenet_v3/mobilenet_v3_backbone_presets_test.py#L37))

Reply:
Done